src/cdk/v2/destinations/pinterest_tag/procWorkflow.yaml

```diff
@@ -129,7 +133,11 @@ steps:
             "num_items": .properties.numOfItems && Number(.properties.numOfItems),
             "order_id": .properties.order_id,
             "search_string": .properties.query,
-            "opt_out_type": .properties.optOutType
+            "opt_out_type": .properties.optOutType,
```

Comment:
Are all the new properties available in ecomm spec ?

Reply:
No. I mapped whatever was available and others in camel case.